### ISSUE TITLE
Can I get the position and orientation from Vive Tracker via OpenVR APIs? 

### ISSUE THREAD

**Issue by felixshing** [ValveSoftware/openvr#1671]
Hello, I want to capture the position and orientation data from [Vive Tracker](https://www.vive.com/us/accessory/tracker3/), which is used for full-body tracking. Can I get them from OpenVR APIs? I know I can get the position and orientation from the headset and controllers via OpenVR, but I am unsure if I can do the same thing to get the data from Vive Tracker

**Comment by Rectus** [comment]
Yes, they are accessible in similar ways, both from IVRCompositor and IVRInput.
